BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -0,0 +1,596 @@
+                    call_vcf_index=SubsetEval.output_vcf_index,
+                    call_output_sample_name=call_output_sample_name,
+                    base_vcf=SubsetTruth.output_vcf,
+                    base_vcf_index=SubsetTruth.output_vcf_index,
+                    base_output_sample_name=base_output_sample_name,
+                    reference=reference,
+                    evaluation_bed=evaluation_bed,
+                    score_field=score_field,
+                    strat_label=stratifier.label,
+                    preemptible=3
+            }
+        }
+
+        scatter (selector in selector_list) {
+            # Use the first outputs for tp_base_vcf, fp_vcf, etc. since these should be the same across all score_fields
+            call BCFToolsStats {
+                input:
+                    tp_base_vcf=StandardVCFEval.tp_base_vcf[0],
+                    tp_base_index=StandardVCFEval.tp_base_index[0],
+                    tp_call_vcf=StandardVCFEval.tp_call_vcf[0],
+                    tp_call_index=StandardVCFEval.tp_call_index[0],
+                    fp_vcf=StandardVCFEval.fp_vcf[0],
+                    fp_index=StandardVCFEval.fp_index[0],
+                    fn_vcf=StandardVCFEval.fn_vcf[0],
+                    fn_index=StandardVCFEval.fn_index[0],
+                    strat_label=select_first([stratifier.label, ""]),
+                    bcf_selector=selector.bcf_selector,
+                    bcf_label=selector.bcf_label,
+                    call_output_sample_name=call_output_sample_name,
+                    base_output_sample_name=base_output_sample_name,
+                    evaluation_bed=evaluation_bed
+            }
+        }
+    }
+
+    call CombineSummaries {
+        input:
+            ROC_summaries=flatten(StandardVCFEval.ROC_summary),
+            SN_summaries=flatten(BCFToolsStats.full_sn),
+            IDD_summaries=flatten(BCFToolsStats.full_idd),
+            ST_summaries=flatten(BCFToolsStats.full_st)
+    }
+
+    output {
+        File simple_summary = CombineSummaries.simple_summary
+        File combined_IDD = CombineSummaries.IDD_combined_summaries
+        File combined_ST = CombineSummaries.ST_combined_summaries
+        File combined_ROC = CombineSummaries.ROC_combined_summaries
+    }
+
+}
+
+task SubsetVCF {
+    input {
+        File input_vcf
+        File input_vcf_index
+        String? input_sample_name
+        Reference reference
+
+        Stratifier stratifier
+        Int interval_padding = 0    # Amount of bases to add around stratification intervals
+
+        String gatk_tag
+        Int? preemptible
+        Int disk_size = 10 + ceil(4.2 * size(input_vcf, "GB") + 2.2 * size(input_vcf_index, "GB") + size(reference.fasta, "GB"))
+        Int cpu = 4
+        Int memory = 16
+    }
+
+    command <<<
+        set -xeuo pipefail
+
+        # Create symlink for VCF & index in case their paths are different, e.g. when using TDR
```

Comment:
oh yikes, that's rough.  Is it possible to specify the index path to gatk instead of needing these symlinks?

Reply:
Unfortunately that doesn't seem to be possible with the current SelectVariants (but maybe coming soon?)